cx_Freeze/hooks.py

```diff
@@ -71,6 +71,11 @@ def initialize(finder):
         finder.ExcludeModule("vms_lib")
     finder.ExcludeModule("new")
     finder.ExcludeModule("Tkinter")
+    #include the _cffi_backend (only if present in system or venv)
+    try:
+        finder.IncludeModule("_cffi_backend")
+    except ImportError:
+        pass
```

Comment:
I'm not excited about always including _cffi_backend. This should be added to the relevant module/package hooks instead, I think -- not global!

Reply:
I just created a hook for cryptography that does this and it seems to work. Please take a look. Thanks!

Reply:
I would not have liked to make this patch like this. But I'll try to explain to you why I chose to do it that way. If you check it out, I made some modifications until I get to this "simpler" version.
But first I will talk about cffi: cffi is a package for compiling modules into other packages. So we'll take cryptography as an example, and it needs cffi to generate some modules, but the modules generated don't need the cffi package, just the _cffi_backend module, which in windows is _cffi_backend.pyd.
So the patch you made solves, but includes an unnecessary package, cffi, and could include only the _cff_backend modules.
But now I will talk because I chose to do the way I did: I use some packages that use cffi: cryptography, bcrypt, Pillow ... so for each case, there would have to be a hook. So if you have 1000 packages using cffi, there will always be an issue about it. Adding globally, I solved the problem not only for all 3 packages, but for any package that uses cffi. And there is one more thing. If the package is not in the system, ie if you do not have a package that uses cffi in venv, it will not be forced to be included.

Reply:
Oh, and in linux, the patch #494 is necessary to include libffi*.so and others in proper path.

Reply:
I've merged #494 and modified the hook for cryptography as you suggested. Thanks! I'm not sure how this PR got closed, but I think its resolved for now. :-)

Reply:
ok, then I'll hook to other known libraries then,